devops/azure-pipelines/build-job-template.yml

```diff
@@ -10,9 +10,9 @@ jobs:
         inputs:
           versionSpec: '${{ parameters.nodeVersion }}'
         displayName: 'Install Node.js'
-      - script: npm ci
-        displayName: 'npm install'
-      - script: npm test -- --watchAll=false
-        displayName: 'npm test'
-      - script: npm run build
-        displayName: 'npm build'
+
+      - script: |
+          npm ci
+          npm run build
+          npm test -- --watchAll=false
```

Comment:
we'd probably prefer these to be separate steps, so we could see by glancing at the run which steps was the cause of the failure, and know how long each one of them is taking separately, etc. also so that, if somehow npm ci or npm run build has a failure exit code while test didn't, the build would still fail.

Reply:
Steps separated again.